fava/ext/portfolio_list/templates/portfolio_list.html

```diff
@@ -0,0 +1,13 @@
+{% extends "_layout.html" %}
```

Comment:
This should be handled by the extension system, the extension should only provide the contents for the content block

Reply:
Actually, I want to make a similar change for the reports that we ship with, so you can wait for that change

Reply:
Done, created a loader function in order to handle template includes of extensions  - https://github.com/beancount/fava/pull/872/commits/da80a9241f1080a1d1fd107f6ed156afd23a70df#diff-6d90976a6c500672baef2abdbc814f8f